Resources/Prototypes/Entities/Objects/Devices/pinpointer.yml

```diff
@@ -54,3 +54,11 @@
   - type: Pinpointer
     updateTargetName: true
     canRetarget: true
+
+- type: entity
+  parent: PinpointerBase
+  id: PinpointerStation
+  components:
+  - type: Pinpointer
+    component: BecomesStation
```

Comment:
I think StationDataComponent would make more sense right?

Reply:
i used that originally but it just broke fsr and would never track it, while this works